src/test/java/org/opentestsystem/ap/imrt/common/repository/BaseItemRepositoryIntegrationTest.java

```diff
@@ -54,15 +54,20 @@ public void shouldSaveAnItemThatHasAttachments() {
         final Attachment aslAttachment = new Attachment(imrtItem,
                 AttachmentFileTypes.ASL,
                 "asl-filename",
-                Instant.now());
+                Instant.now(),
+                "me");
```

Comment:
My lack of understanding some Spring magic is showing again here.  I found that if I didn't set the `updatedBy` field "manually", then the value wouldn't be set.  I suppose this constructor could actually be the culprit - maybe I should ditch this constructor and just use the getters/setters to get/set each individual field.